buildarm.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+export ARCH=arm
+TARGET=$1
+
+# Sets up toolchain environment variables for arm toolchain
+# Tested against armv7-eabihf, glibc from toolchains.free-electrons.com
+
+warn()
+{
+	echo "$1" >&2
+}
+
+if [ ! -z $(which arm-linux-gcc-5.4.0) ];
```

Comment:
can we get rid of the hard-coded version? there should be a symlink for a non-versioned name of the binary like `arm-linux-gcc` or something